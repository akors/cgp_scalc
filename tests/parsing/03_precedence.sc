2 + (3 * 5)
2 + 3 * 5
(2 + 3) * 5
1 + 99 / 9 * 7
1 + 99 / (9 * 7)
(1 + 99) / (9 * 7)
1 + ( 99 / (9 * (1 + 7)) )
2 + 3 - 1 / (2*3)^(1/2)
12 - -9
