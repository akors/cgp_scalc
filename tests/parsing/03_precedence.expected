17
17
25
78
2.57143
1.5873
2.375
4.59175
21
